### ISSUE TITLE
Qwik properties use a kebab-case

### ISSUE THREAD

**Issue by dejurin** [antfu-collective/icones#320]
### Additional context

For example: stop-color insted stopColor.

<img width="1051" alt="Screenshot 2024-05-27 at 17 53 55" src="https://github.com/antfu-collective/icones/assets/4470015/5868553a-bd17-4abc-9686-af25ca7ec756">

**Comment by dejurin** [comment]
For example, you can see for yourself by looking at an example of how to use SVG       `<stop offset="1" stop-color="#ca8a04" />` here https://github.com/fabian-hiller/valibot/blob/main/website/src/icons/LogoIcon.tsx

<img width="674" alt="Screenshot 2024-05-27 at 18 26 21" src="https://github.com/antfu-collective/icones/assets/4470015/19538e9c-4611-4436-b5e5-132f2d3cb057">

<img width="707" alt="Screenshot 2024-05-27 at 18 26 31" src="https://github.com/antfu-collective/icones/assets/4470015/84ad0e95-91d5-4e3d-ad3c-744c149ba404">

**Comment by cyberalien** [comment]
Forgot that `gradientTransform` is always camel case.

Thanks for screenshot, now I see what's the issue. React JSX uses camelCase for everything, including `stopColor`, Qwik works with unmodified SVG.

So issue here is React JSX transforms should not be applied to Qwik code samples.

**Comment by dejurin** [comment]
> Forgot that `gradientTransform` is always camel case.
> 
> Thanks for screenshot, now I see what's the issue. React JSX uses camelCase for everything, including `stopColor`, Qwik works with unmodified SVG.
> 
> So issue here is React JSX transforms should not be applied to Qwik code samples.

Yep, Qwik use more natively properties. For example in React you use className in Qwik you use class .

**Comment by antfu** [comment]
PR welcome to fix it :)
